examples/examples.js

```diff
@@ -157,6 +157,44 @@ const journalsConfig = {
 	}
 };
 
+const identityAdminLoggedOutConfig = {
+	filename: "identityadmin-loggedout",
+	title: "Identity Admin Logged out",
+	global_nav_config: {
+		service: "identitymanagement",
```

Comment:
service should just be empty here as we don't have an identitymanagement link in the TopHat?